spec/unit/sync-accumulator.spec.js

```diff
@@ -0,0 +1,274 @@
+"use strict";
+import 'source-map-support/register';
+const sdk = require("../..");
```

Comment:
New code should probably be using `import` (and definitely not a mixture of both)